content/docs/observability/deployment/online.md

```diff
@@ -121,17 +118,23 @@ To perform an online installation of CSM for Observability, the following steps
     |
     |- Creating namespace karavi                                        Success
     |
+    |- CSI Driver for PowerFlex is installed                            Success
+    |
     |- Copying Secret from vxflexos to karavi                           Success
     |
-    |- Installing CertManager CRDs                                      Success
+    |- CSI Driver for PowerStore is installed                           Success
     |
-    |- Installing Karavi Observability helm chart                       Success
+    |- Copying Secret from powerstore to karavi                         Success
     |
-    |- Waiting for pods in namespace karavi to be ready                 Success
+    |- Installing CertManager CRDs                                      Success
     |
-    |- Copying Secret from vxflexos to karavi                           Success
+    |- Enabling Karavi Authorization for Karavi Observability
+      |
+      |--> Copying ConfigMap from vxflexos to karavi                    Success
+      |
+      |--> Copying Karavi Authorization Secrets from vxflexos to karavi Success
     |
-    |- Enabling Karavi Authorization for Karavi Observability           Success
+    |- Installing Karavi Observability helm chart                       Success
```

Comment:
CSM Observability?

Reply:
These references to Karavi will be addressed at a later time when we do a full name change of our repositories